bitnami/redis/Chart.yaml

```
@@ -34,4 +34,4 @@ maintainers:
 name: redis
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/redis
-version: 18.16.0
+version: 18.16.1
```

Please note you're adding a new parameter, therefore you must bump minor version instead:
```suggestion
version: 18.17.0
```